cli/pipeline/template.go

```diff
@@ -132,7 +132,7 @@ spec:
         - "-c"
         - "mkdir -p $INPUT_PATH $OUTPUT_PATH &&
           {{ range $index, $input := .Step.Inputs }}
-          paddle data get {{ $input.Step }}/{{ $input.Version }} $INPUT_PATH -b {{ $input.Branch | sanitizeName }} -p {{ $input.Path }} {{ $input.Bucket | bucketParam }} &&
+          paddle data get {{ $input.Step }}/{{ $input.Version }} $INPUT_PATH -b {{ $input.Branch | sanitizeName }} -p {{ $input.Path }} -d {{ .Step.Subdir }} {{ $input.Bucket | bucketParam }} &&
```

Comment:
This doesn't seem to work:
```
panic: template: podTemplate:97:147: executing "podTemplate" at <.Step.Subdir>: can't evaluate field Subdir in type string
```
Not sure how best to cast bool to string here?

Reply:
I think it needs to be `$input.Subdir`. The line above looks like it's basically doing a for loop over each element in `Step.Inputs` and binding it to `$input`. 

I've not used this templating library before, but the docs are here in case you haven't seen them: https://godoc.org/text/template